### ISSUE TITLE
val_dataset is Test_dataset?

### ISSUE THREAD

**Issue by WYC-321** [jeya-maria-jose/Medical-Transformer#79]
Hello, the paper only has the division information of the training set and the test set, and does not mention the validation set. I don't know how to get the final model. Get the metrics on the test set every 10 epochs during training, and use the best results as the reported results in the paper after 400 epochs?

**Comment by jeya-maria-jose** [comment]
Yes, we test set every 10 epochs during training, and use the best results as the reported results after 400 epochs.
